tests/components/aemet/test_weather.py

```diff
@@ -42,10 +42,10 @@ async def test_aemet_weather(hass):
     assert state.state == ATTR_CONDITION_SNOWY
     assert state.attributes.get(ATTR_ATTRIBUTION) == ATTRIBUTION
     assert state.attributes.get(ATTR_WEATHER_HUMIDITY) == 99.0
-    assert state.attributes.get(ATTR_WEATHER_PRESSURE) == 100440.0
+    assert state.attributes.get(ATTR_WEATHER_PRESSURE) == 1004.4  # 100440.0 Pa -> hPa
```

Comment:
It's because of these changes:
> The PR also aligns default units for pressure and wind speed with the documentation:
> - If unit system is metric, default pressure unit is hPa and the default wind speed unit is km/h
> - If unit system is imperial, default pressure unit is inHg and the default wind speed unit is mi/h

We could introduce those changes in a separate PR instead, merge that and then rebase this one?

Reply:
Shouldn't we set a unit for these integrations so that we don't break anything?

Reply:
The problem today is that:
- if integrations do set a unit for `pressure`, the pressure is converted to Pa if unit system is set to metric and to psi if unit system is set to imperial. That's contrary to the documentation which stipulates `hPa` / `inHg`
- if integrations do set a unit for `wind_speed`, the wind_speed is converted to `m/s` if unit system is set to metric and to `mi/h` if unit system is set to imperial. That's contrary to the documentation which stipulates `km/h` / `mi/h`

Reply:
Ok, but then we need to explain the breaking change in the breaking change paragraph.

Reply:
Should we do that here (keep everything in one place) or do we open a PR for specifically this unit conversion with a breaking change?

Reply:
Done. Needs to add the breaking change label.